internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/PushStatusDecider.java

```diff
@@ -63,14 +64,16 @@ public Pair<ExecutionStatus, Optional<String>> checkPushStatusAndDetails(
       int replicationFactor = pushStatus.getReplicationFactor();
       int errorReplicasCount = partition.getErrorInstances().size();
       int completedReplicasCount = partition.getReadyToServeInstances().size();
-      int assignedReplicasCount = partition.getAllInstances().size();
-      logger.debug(
-          "Checking Push status for offline push for topic: {} Partition: {} has {} assigned replicas including {} error replicas, {} completed replicas.",
-          pushStatus.getKafkaTopic(),
-          partition.getId(),
-          assignedReplicasCount,
-          errorReplicasCount,
-          completedReplicasCount);
+      if (logger.isDebugEnabled()) {
```

Comment:
Is this check necessary if we use lambda for ` partition.getNumOfTotalInstances()`?

Reply:
Lambda could be an alternative way to do it, but it would instantiate a lambda object, which isn't necessary IMHO...